lib/foreman_rh_cloud/engine.rb

```diff
@@ -157,37 +157,37 @@ def self.register_scheduled_task(task_class, cronline)
       end
     end
 
-    initializer 'foreman_rh_cloud.register_rex_features', :before => :finisher_hook do |_app|
+    config.to_prepare do
```

Comment:
Small nitpick: can we have all initializers together and then the `to_prepare` block?

Reply:
Sure